ts/server.go

```diff
@@ -89,8 +89,9 @@ func (s *Server) handleQuery(w http.ResponseWriter, r *http.Request, _ httproute
 		response.Results = append(response.Results, &TimeSeriesQueryResponse_Result{
 			Name:             q.Name,
 			Sources:          sources,
-			SourceAggregator: q.SourceAggregator,
-			Downsampler:      q.Downsampler,
+			SourceAggregator: q.GetSourceAggregator().Enum(),
```

Comment:
If the response had sent "nil" for any for these, the response would also contain nil. This style reflects back what the server used, even if the server used a default.